pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -733,7 +748,7 @@ public static class Accounting {
     public static final String CONFIG_OF_ALARMING_LEVEL_HEAP_USAGE_RATIO = "accounting.oom.alarming.usage.ratio";
     public static final float DEFAULT_ALARMING_LEVEL_HEAP_USAGE_RATIO = 0.75f;
 
-    public static final String CONFIG_OF_HEAP_USAGE_PUBLISH_PERIOD = "accounting.heap.usage.publish.period";
+    public static final String CONFIG_OF_HEAP_USAGE_PUBLISHING_PERIOD = "accounting.heap.usage.publishing.period";
```

Comment:
just to clarify - we publish every 5 secs ? 

you might want to include the unit (millis or whatever) as a suffix in the config name key

Reply:
done